deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -101,3 +108,93 @@ def make_vuln_callback(pattern):
     regex = re.compile(r'{}{}'.format(vulnerability_detector_prefix, pattern))
 
     return lambda line: regex.match(line) is not None
+
+
+def load_db(db_path):
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+    return conn, cursor
+
+
+def make_query(db_path, query_list):
+    """
+    Makes a query to the database in db_path for each item in query_list
+    """
+    try:
+        db = load_db(db_path)
+        for item in query_list:
+            db[1].execute(item)
+        db[0].commit()
+    except sqlite3.Error as error:
+        print("Failed perform sqlite query\n", error)
+        raise error
+    finally:
+        db[1].close()
+        db[0].close()
+
+
+
+def insert_package_in_db(agent = "000", format = "rpm", name = "WazuhIntegrationPackage", 
+    priority = "", section = "Unspecified", size= 99, vendor = "WazuhIntegrationTests", version = "1.0.0-1.el7",
+    architecture = "x86_64", multiarch = "", description = "Wazuh Integration tests mock package", 
+    source = "Wazuh Integration tests mock package", location = "", triaged = 0, cpe = "",
+    install_time = datetime.datetime.now().strftime("%Y/%m/%d %H:%M:%S")
+):
+    """
+    Insert a new package in the installed package database, with the parameters given as arguments.
+    If used in conjunction with insert_vulnerability using the default arguments it will generate an alert.
+    """
+    path =  os.path.join(PACKAGES_DB_PATH,"{}.db".format(agent))
+    query_string = """INSERT INTO sys_programs
+        (scan_id, scan_time, format, name, priority, section, size, vendor, install_time, version, \
+            architecture, multiarch, source, description, location, triaged, cpe)
+        select scan_id, scan_time, \"{}\", \"{}\", \"{}\", \"{}\", \"{}\", \"{}\", 
+        \"{}\", \"{}\", \"{}\",  \"{}\", \"{}\", \"{}\", \"{}\", \"{}\", \"{}\"
+        from sys_programs limit 1
+        """.format(
+            format, name, priority, section, size, vendor, install_time, version, architecture, 
+            multiarch, source, description, location, triaged, cpe
+        )
```

Comment:
I think it would be better to change this to `fstrings`. Also, the strings from the functions `insert_vulnerability` and `update_package`.

Reply:
Done in ff83ea8